src/Client.php

```diff
@@ -182,6 +196,22 @@ public function setDefaultChannel($channel)
     }
 
     /**
+     * @return bool
+     */
+    public function isStickyChannel()
+    {
+        return $this->sticky_channel;
```

Comment:
This will force the result to be an boolean. Otherwise it can also be any other type.
`return (bool) $this->sticky_channel;`